apps/etterlatte-oppdater-behandling/src/main/kotlin/OmberegningHendelser.kt

```diff
@@ -0,0 +1,53 @@
+package no.nav.etterlatte
+
+import com.fasterxml.jackson.module.kotlin.treeToValue
+import io.ktor.client.call.body
+import kotlinx.coroutines.runBlocking
+import no.nav.etterlatte.libs.common.behandling.Omberegningshendelse
+import no.nav.etterlatte.libs.common.logging.withLogContext
+import no.nav.etterlatte.libs.common.objectMapper
+import no.nav.etterlatte.libs.common.rapidsandrivers.correlationId
+import no.nav.etterlatte.libs.common.rapidsandrivers.eventName
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.MessageContext
+import no.nav.helse.rapids_rivers.RapidsConnection
+import no.nav.helse.rapids_rivers.River
+import org.slf4j.LoggerFactory
+import java.util.*
+
+const val hendelsetype = "OMBEREGNINGHENDELSE"
+
+internal class OmberegningHendelser(rapidsConnection: RapidsConnection, private val behandlinger: Behandling) :
+    River.PacketListener {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    init {
+        logger.info("initierer rapid for omberegninghendelser")
+        River(rapidsConnection).apply {
+            eventName("OMBEREGNINGHENDELSE")
```

Comment:
Skulle vi trekt ut denne som variabel også? Fra da vi hadde R&R tidligere var det veldig vanskelig å oppdage hvem som consuma de forskjellige topicsa. Kunne vært en idè å tilrettelegge for noe nå som vi drar i gang med noe Kafka igjen?